crates/cli/src/format.rs

```diff
@@ -359,4 +516,73 @@ main =
 
         cleanup_temp_dir(dir);
     }
+
+    const HEADER: &str = indoc! {r#"
+        app [main] { pf: platform "https://github.com/roc-lang/basic-cli/releases/download/0.15.0/SlwdbJ-3GR7uBWQo6zlmYWNYOxnvo8r6YABXD-45UOw.tar.br" }
```

Comment:
I've used basic-cli as the platform in tests but this would then need to be manually updated with new releases and breaking changes can't be pushed without updating it. The problem is that the tests need to be done in a temp directory as they update the files when run. Should it instead copy a platform into the temp directory or find the absolute path to a some platform (maybe in the examples directory?).

Reply:
There's a zig platform in that crate used for tests. I'd suggest using that if we can.